src/screens/SearchScreen.js

```diff
@@ -17,17 +17,26 @@ import {map, navKeys} from '../constants';
 
 class SearchScreen extends Component {
     componentWillMount() {
-        const {lastKnownRegion, initializeMap} = this.props;
+        const {lastKnownRegion} = this.props;
 
-        initializeMap(lastKnownRegion);
+        this.props.initializeMap(lastKnownRegion);
     }
 
     componentDidMount() {
-        const {lastKnownRegion, navigation: {setParams}, mapHasLoaded} = this.props;
+        const {lastKnownRegion, navigation: {setParams}} = this.props;
 
         setParams({buttonName: map.SearchOptions.LIST, toggleButton: this.toggleButton});
 
-        mapHasLoaded();
+        this.props.mapHasLoaded();
```

Comment:
Again, changed the `mapHasLoaded` that I destructured off props to `this.props.mapHasLoaded` so that we're sure at a quick glance that we're correctly doing it off props.